srcpkgs/osmin/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'osmin'
+pkgname=osmin
+version=1.8.7
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_C_COMPILER=/usr/bin/clang -DCMAKE_CXX_COMPILER=/usr/bin/clang++ ."
+hostmakedepends="qt5-qmake pkg-config qt5-host-tools git"
+makedepends="qt5-declarative-devel qt5-3d clang libgomp-devel
+	libomp-devel qt5-declarative-devel qt5-tools-devel
+	qt5-svg-devel   qt5-quickcontrols2-devel qt5-sensors-devel
+	qt5-multimedia-devel qt5-remoteobjects-devel qt5-location-devel"
+short_desc="GPS Navigator On-Road/Off-Road"
+
+archs="~armv6l-musl ~armv7l ~i686" # libomp-devel
+license="GPL-3.0-only"
+maintainer="Eloi Torrents <eloitor@disroot.org>"
+homepage="https://github.com/janbar/osmin"
+_libosmscout_version=1.9.2
+distfiles="
```

Comment:
for this and checksum, put the first distfile/checksum on the same line as the opening `"`